bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="wundergroundupdatereceiver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>Wunderground Update Receiver Binding</name>
+	<description>
+		This binding enables acting as a receiver of updates from weather stations that post measurements to
+		https://rtupdate.wunderground.com/weatherstation/updateweatherstation.php.
+	</description>
+	<author>Daniel Demus</author>
```

Comment:
The author tag is deprecated and should therefore be removed. See openhab/openhab-core#1844.